### PR TITLE
Fix type error in `downloads` content element

### PR DESCRIPTION
**Pull request by fritzmg**
If you create a `downloads` content element, but then immediately click "Back", the following error will occur:

```
TypeError:
Contao\CoreBundle\Filesystem\FilesystemUtil::listContentsFromSerialized(): Argument #2 ($sources) must be of type array|string, null given, called in vendor\contao\contao\core-bundle\src\Controller\ContentElement\DownloadsController.php on line 86

  at vendor\contao\contao\core-bundle\src\Filesystem\FilesystemUtil.php:37
  at Contao\CoreBundle\Filesystem\FilesystemUtil::listContentsFromSerialized()
     (vendor\contao\contao\core-bundle\src\Controller\ContentElement\DownloadsController.php:86)
  at Contao\CoreBundle\Controller\ContentElement\DownloadsController->getFilesystemItems()
     (vendor\contao\contao\core-bundle\src\Controller\ContentElement\DownloadsController.php:41)
  at Contao\CoreBundle\Controller\ContentElement\DownloadsController->getResponse()
     (vendor\contao\contao\core-bundle\src\Controller\ContentElement\AbstractContentElementController.php:40)
  at Contao\CoreBundle\Controller\ContentElement\AbstractContentElementController->__invoke()
     (vendor\contao\contao\core-bundle\src\Controller\ContentElement\AbstractDownloadContentElementController.php:40)
  at Contao\CoreBundle\Controller\ContentElement\AbstractDownloadContentElementController->__invoke()
     (vendor\symfony\http-kernel\HttpKernel.php:181)
  at Symfony\Component\HttpKernel\HttpKernel->handleRaw()
     (vendor\symfony\http-kernel\HttpKernel.php:76)
  at Symfony\Component\HttpKernel\HttpKernel->handle()
     (vendor\symfony\http-kernel\HttpCache\SubRequestHandler.php:86)
  at Symfony\Component\HttpKernel\HttpCache\SubRequestHandler::handle()
     (vendor\symfony\http-kernel\Fragment\InlineFragmentRenderer.php:78)
  at Symfony\Component\HttpKernel\Fragment\InlineFragmentRenderer->render()
     (vendor\symfony\http-kernel\Fragment\FragmentHandler.php:83)
  at Symfony\Component\HttpKernel\Fragment\FragmentHandler->render()
     (vendor\contao\contao\core-bundle\src\Fragment\FragmentHandler.php:66)
  at Contao\CoreBundle\Fragment\FragmentHandler->render()
     (vendor\contao\contao\core-bundle\contao\elements\ContentProxy.php:74)
  at Contao\ContentProxy->generate()
     (vendor\contao\contao\core-bundle\contao\library\Contao\Controller.php:601)
  at Contao\Controller::getContentElement()
     (vendor\contao\contao\core-bundle\contao\dca\tl_content.php:1007)
  at tl_content->addCteType()
     (vendor\contao\contao\core-bundle\contao\drivers\DC_Table.php:5014)
  at Contao\DC_Table->parentView()
     (vendor\contao\contao\core-bundle\contao\drivers\DC_Table.php:468)
  at Contao\DC_Table->showAll()
     (vendor\contao\contao\core-bundle\contao\classes\Backend.php:546)
  at Contao\Backend->getBackendModule()
     (vendor\contao\contao\core-bundle\contao\controllers\BackendMain.php:144)
  at Contao\BackendMain->run()
     (vendor\contao\contao\core-bundle\src\Controller\BackendController.php:44)
  at Contao\CoreBundle\Controller\BackendController->mainAction()
     (vendor\symfony\http-kernel\HttpKernel.php:181)
  at Symfony\Component\HttpKernel\HttpKernel->handleRaw()
     (vendor\symfony\http-kernel\HttpKernel.php:76)
  at Symfony\Component\HttpKernel\HttpKernel->handle()
     (vendor\symfony\http-kernel\Kernel.php:197)
  at Symfony\Component\HttpKernel\Kernel->handle()
     (public\index.php:42)           
```

### CODE REVIEW

File: core-bundle/src/Controller/ContentElement/DownloadsController.php [hunk 1]
```diff
@@ -83,7 +83,7 @@ protected function getFilesystemItems(Request $request, ContentModel $model): Fi
         };
 
         // Find filesystem items
-        $filesystemItems = FilesystemUtil::listContentsFromSerialized($this->filesStorage, $sources);
+        $filesystemItems = FilesystemUtil::listContentsFromSerialized($this->filesStorage, $sources ?? []);
```
Comment (m-vo) [inline core-bundle/src/Controller/ContentElement/DownloadsController.php]: I think I would move this up to L82.

Comment (fritzmg) [reply core-bundle/src/Controller/ContentElement/DownloadsController.php]: Or should we allow `null` for `listContentsFromSerialized` instead? Serialized fields in Contao are always nullable.

Comment (m-vo) [reply core-bundle/src/Controller/ContentElement/DownloadsController.php]: I'm fine with this as well. 👍

**Review by m-vo — APPROVED** [review]

**Review by leofeyer — APPROVED** [review]

**Comment by leofeyer** [comment]
Thank you @fritzmg.

**Pull request merged**